client/app/src/main/java/com/microsoft/research/karya/ui/consentForm/ConsentFormActivity.kt

```diff
@@ -20,7 +20,7 @@ import com.microsoft.research.karya.ui.registration.RegistrationActivity
 import kotlinx.android.synthetic.main.activity_consent_form.*
 import kotlinx.coroutines.launch
 
-class ConsentFormActivity : BaseActivity(useAssistant = true) {
+class ConsentFormActivity : BaseActivity() {
```

Comment:
This activity will be removed with my PR so I don't see any point in changing it.

Reply:
Sure, I changed it because hilt doesn't support this. I will change things after your PR. This was done so that I can test my implementation